tensorflow/contrib/rnn/python/kernel_tests/core_rnn_test.py

```diff
@@ -2205,8 +2205,7 @@ def _execute_rnn_on(self,
     return run_metadata
 
   def testRNNOnCPUCellOnGPU(self):
-    # Currently not implemented for OpenCL
-    if not test.is_gpu_available() or test_util.is_sycl_enabled():
```

Comment:
Can that be done as a separate commit? This already has been reverted in the opencl_improvements branch and will cause merge conflict